docs/tizen-studio/common-tools/device-manager.md

```diff
@@ -0,0 +1,223 @@
+Monitoring Devices with the Device Manager
+==========================================
+
+## Dependencies
+
+- Tizen Studio 2.0 and Higher
+
+The Device Manager is a standalone tool that manages devices and allows you to access their internal file systems and logs. A device can be either a virtual device, such as an emulator, or a physical device connected with a USB cable or a wireless connection.
+
+The Device Manager provides a graphical user interface that is more convenient than the CLI (Command Line Interface). You can use it to connect and disconnect devices to and from your computer, as well as to access devices' internal file systems and push or pull files between the device and your computer. The Device Manager provides the device logs in a table format, and you can use filters to find the entries that interest you.
+
+To use the Device Manager with a specific device, you must connect the device over SDB (Smart Development Bridge). Physical devices can be connected with a USB cable, and emulators can be connected through the Emulator Manager. When the device is connected over SDB, it is automatically displayed in the Device Manager and its directory structure and logs are shown in the **File Explorer** and **Log** views, respectively.
+
+**Note** If you do not have the Device Manager installed, open the [Package Manager](../download/updating_sdk.htm), go to the **Main SDK** tab, and locate the **Device Manager** under **Tizen SDK Tools > Baseline SDK**. To install the tool, click **Install**.
+
+
+Accessing Device Manager {#access}
```

Comment:
I think you have to change the following:
From:
Accessing Device Manager {#access}

To:
<a name="access"></a>
## Accessing Device Manager 

There are this kinds of changes. Please update and resend a new pull request.